practice-three/src/css/style.css

```diff
@@ -51,6 +51,11 @@
   padding: 10px 85px;
   background-color: var(--primary-color-100);
   color: var(--primary-color-200);
+  cursor: pointer;
+}
+
+.btn-table {
```

Comment:
Not a suitable name. Should refer here: https://getbootstrap.com/docs/5.2/components/buttons/